Tribler/Test/Core/Modules/MetadataStore/test_store.py

```diff
@@ -5,10 +5,11 @@
 import string
 from binascii import unhexlify
 
+
```

Comment:
Is this additional newline intended?

Reply:
No. I didn't run my pre-commit hooks on this thing, see...